File: Day 02 - Operators/solution.js
// Complete the solve function below.
function solve(meal_cost, tip_percent, tax_percent) {
  //meal_cost= 12;
  //tip_percent = 20;
  //tax_percent =8;
  var tip = (meal_cost * tip_percent) / 100;
  var tax = (meal_cost * tax_percent) / 100;
  var totalCost = Math.round(meal_cost + tip + tax);
  console.log(totalCost);
}

File: Day 02 - Operators/solution.py
# Complete the solve function below.


def solve(meal_cost, tip_percent, tax_percent):
    tip = (meal_cost * tip_percent) / 100
    tax = (meal_cost * tax_percent) / 100
    total_coast = meal_cost + tip + tax
    return total_coast


if __name__ == '__main__':
    meal_cost = float(input())
    tip_percent = int(input())
    tax_percent = int(input())
    total_coast = round(solve(meal_cost, tip_percent, tax_percent))
    print(total_coast)
